partcad/tests/unit/test_enrich_resolution.py

```
@@ -0,0 +1,137 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)